edx_repo_tools/conventional_commits/extra-py312.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.12
 # by the following command:
 #
-#    pip-compile --output-file=edx_repo_tools/conventional_commits/extra-py312.txt edx_repo_tools/conventional_commits/extra-py312.in
```

Comment:
I think you need to run `make upgrade` so that your personal file path is not in the output files.

Reply:
done